.github/workflows/mike.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
     - name: Create new version
       run: |
         git config --global user.name "Srinivasan Parthasarathy"
-        git config --global user.email 5309125+sriumcp@users.noreply.github.com   
+        git config --global user.email spartha@us.ibm.com  
```

Comment:
No objection but perhaps use the iter8-tools email?

Reply:
Yes, agreed. We should use the iter8.tools email... can handle in a subsequent PR.